concourse/scripts/create-cf.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+
+SCRIPT_DIR=$(cd "$(dirname $0)" && pwd)
+ATC_URL=${ATC_URL:-"http://192.168.100.4:8080"}
+FLY_TARGET=${FLY_TARGET:-$ATC_URL}
+
+env=${DEPLOY_ENV-$1}
+pipeline="deploy-cloudfoundry"
+config="${SCRIPT_DIR}/../pipelines/create-cf.yml"
+bosh_password=$("$SCRIPT_DIR"/s3curl.pl --id="$AWS_ACCESS_KEY_ID" --key="$AWS_SECRET_ACCESS_KEY" --  \
```

Comment:
Yes, that's a temporary solution until [Generate secrets automatically in an environment](https://www.pivotaltracker.com/n/projects/1275640/stories/110059806) is done. I'll make a note in Pivotal.